merlin_models/loader/backend.py

```diff
@@ -27,6 +27,8 @@
 except ImportError:
     cp = np
 
+from merlin.graph.tags import Tags
```

Comment:
I would prefer it if this would be moved to either the top-level `from merlin import Tags` or to core: `from merlin.core import Tags`

Reply:
Agreed - I was thinking we should go `from merlin.schema import Schema, Tags`

Reply:
I'm good with any of those. We've dragged our feet on shortening imports on the NVT side, but anything we use in multiple libraries should be quick and clean to import 👍🏻

Reply:
I've moved here https://github.com/NVIDIA-Merlin/core/pull/11 (also made is_list and is_ragged public)

Reply:
made that update -